docs/topics/modules/approving_build_pipeline.adoc

```diff
@@ -3,34 +3,40 @@
 
 Several minutes after starting, your build pipeline is ready to be promoted to `Stage`.
 
-The build pipeline includes steps for a `Stage` build and a `Run` build. `Stage` is a production staging area to review and test changes before they finalizing them and staging the build on `Run`.
+The build pipeline includes steps for a `Stage` build and a `Run` build. `Stage` is a production staging area to review and test your application before finalizing it and staging the build on `Run`.
```

Comment:
So, "staging to Stage, which is a staging area" makes sense, but I don't think we shoud then say "staging the build on Run". That's confusing. Can we say "deploying the build on Run"?